S-L-Murphey/m-aqua
# Slm added js

Martin added hundreds more fish to include. Added a js file to simplify future additions.

## Steps to Test

1. Pull this branch
2. Open in chrome
3. Verify that a JS file to added under the folder scripts
